packages/gatsby-plugin-utils/src/polyfill-remote-file/graphql/gatsby-image-resolver.ts

```diff
@@ -241,6 +241,12 @@ export async function gatsbyImageResolver(
     }
   }
 
+  if (
+    [`true`, `1`].includes(process.env.GATSBY_SHOULD_TRACK_IMAGE_CDN_URLS || ``)
+  ) {
+    actions.addGatsbyImageSourceUrl(source.url)
```

Comment:
Please do check if `addGatsbyImageSourceUrl` is available before trying to use it. Otherwise we might see crashes when older gatsby version is used with newer `gatsby-plugin-utils`.

Reply:
Good idea - fixed!